shared/i18n/locales/zh_TW/translation.json

```diff
@@ -1016,9 +1016,13 @@
   "Create a \"Web\" stream in your Google Analytics admin dashboard and copy the measurement ID from the generated code snippet to install.": "在 Google Analytics 管理儀表板中創建 \"Web\" 流，並從生成的程式碼片段中複製 measurement ID 進行安裝。",
   "Configure a Matomo installation to send views and analytics from the workspace to your own Matomo instance.": "Configure a Matomo installation to send views and analytics from the workspace to your own Matomo instance.",
   "Instance URL": "Instance URL",
-  "The URL of your Matomo instance. If you are using Matomo Cloud it will end in matomo.cloud/": "The URL of your Matomo instance. If you are using Matomo Cloud it will end in matomo.cloud/",
-  "Site ID": "Site ID",
-  "An ID that uniquely identifies the website in your Matomo instance.": "An ID that uniquely identifies the website in your Matomo instance.",
+  "The URL of your Matomo instance. If you are using Matomo Cloud it will end in matomo.cloud/": "你的 Matomo 實例 URL。如果你使用 Matomo 雲，它將在 matomo.cloud/ 中結束。",
```

Comment:
Remove these changes as the file is auto-generated from [CrowdIn](https://translate.getoutline.com/)